wurst/_handles/Unit.wurst

```diff
@@ -612,3 +633,35 @@ public function unit.incAbilityLevel(int abilityId)
 
 public function unit.decAbilityLevel(int abilityId)
 	DecUnitAbilityLevel(this, abilityId)
+	
+/* Iterator */
```

Comment:
I think regarding comments/doc, either write something useful, or nothing at all. This one doesn't really seem useful